packages/tools/devtools/devtools-test-app/src/FluidObject.ts

```diff
@@ -179,39 +179,56 @@ export class AppData extends DataObject {
 		const builder = new SchemaFactory("DefaultVisualizer_SharedTree_Test");
 
 		// TODO: Maybe include example handle
+		class LeafNodeSchema extends builder.object("leaf-node", {
```

Comment:
High-level feedback: can we make this tree a bit more semantic? As an example case, it would be useful to demonstrate the concepts we're testing here in a way that we would expect consumers to use them.

E.g., we could craft a schema that represents a contact book, or a family tree or something. Then we could (in the future) even develop visuals for the sample app that actually use the tree we've defined.